docs/conf.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'pulp-2to3-migration'
+copyright = '2019, Pulp Team'
+author = 'Pulp Team'
+
+# The short X.Y version
+version = ''
```

Comment:
I think you need to have this imported from the package. See this as an example: https://github.com/pulp/pulp_ansible/blob/master/docs/conf.py#L32-L35